### PR TITLE
feat: 80 characters wide for git commits

### PR DESCRIPTION
**Pull request by chooban**

### CODE REVIEW

File: cows/al.cow [hunk 1]
```diff
@@ -5,84 +5,84 @@ $the_cow = <<EOC;
...
-%&&&&&&%&&&&&%%%%%&%&&&&&&%%&&&%&&&&&&&&&&&&&&%%&&&&&&&&&%&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&%&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&
-%&&&&&&&%&&%%%%%%%&&&%%%%%%%&%&&&&&&&&&&&&&&&%&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&%&&&&&&&&&&&&&&&&&&&&&&&&&%&&&&&&&&&&&&&&&&&&&
-&&&&&&%&&%%%%%%%&%&&&&%%&&&%&&&&&&&&&&&&&&&%&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&@&&&&&
+                                                                                
+                                                                                
+                                                                                
+                                                                                
+                                                                                
+                                                                                
+                                                                      .,,,,,,,,,
```
Comment (AlMcKinlay) [inline cows/al.cow:14]: This line appears to be truncated rather than shrunk

File: cows/al.cow [hunk 2]
```diff
@@ -5,84 +5,84 @@ $the_cow = <<EOC;
...
+                                                                 .,,*,,,,,,,,,,,
```
Comment (AlMcKinlay) [inline cows/al.cow:15]: This line appears to be truncated rather than shrunk

File: cows/al.cow [hunk 3]
```diff
@@ -5,84 +5,84 @@ $the_cow = <<EOC;
...
+                                                             .,,,,,,,,,,,,,,,,,,
```
Comment (AlMcKinlay) [inline cows/al.cow:16]: This line appears to be truncated rather than shrunk

File: cows/al.cow [hunk 4]
```diff
@@ -5,84 +5,84 @@ $the_cow = <<EOC;
...
+                                                           .,,,,,.,...,,,.,...,,
```
Comment (AlMcKinlay) [inline cows/al.cow:17]: This line appears to be truncated rather than shrunk

File: cows/al.cow [hunk 5]
```diff
@@ -5,84 +5,84 @@ $the_cow = <<EOC;
...
+                                                         .,....................,
```
Comment (AlMcKinlay) [inline cows/al.cow:18]: This line appears to be truncated rather than shrunk

File: cows/al.cow [hunk 6]
```diff
@@ -5,84 +5,84 @@ $the_cow = <<EOC;
...
+                                                       ........................,
```
Comment (AlMcKinlay) [inline cows/al.cow:19]: This line appears to be truncated rather than shrunk

File: cows/al.cow [hunk 7]
```diff
@@ -5,84 +5,84 @@ $the_cow = <<EOC;
...
+                                                     ...........................
```
Comment (AlMcKinlay) [inline cows/al.cow:20]: This line appears to be truncated rather than shrunk

File: cows/al.cow [hunk 8]
```diff
@@ -5,84 +5,84 @@ $the_cow = <<EOC;
...
+                                                    ,...........................
```
Comment (AlMcKinlay) [inline cows/al.cow:21]: This line appears to be truncated rather than shrunk

File: cows/al.cow [hunk 9]
```diff
@@ -5,84 +5,84 @@ $the_cow = <<EOC;
...
+                                                  ..............................
```
Comment (AlMcKinlay) [inline cows/al.cow:22]: This line appears to be truncated rather than shrunk

File: cows/al.cow [hunk 10]
```diff
@@ -5,84 +5,84 @@ $the_cow = <<EOC;
...
+                                                 .,.............................
```
Comment (AlMcKinlay) [inline cows/al.cow:23]: This line appears to be truncated rather than shrunk

File: cows/al.cow [hunk 11]
```diff
@@ -5,84 +5,84 @@ $the_cow = <<EOC;
...
+                                                ,,..............................
```
Comment (AlMcKinlay) [inline cows/al.cow:24]: This line appears to be truncated rather than shrunk

File: cows/al.cow [hunk 12]
```diff
@@ -5,84 +5,84 @@ $the_cow = <<EOC;
...
+                                               ,,...............................
```
Comment (AlMcKinlay) [inline cows/al.cow:25]: This line appears to be truncated rather than shrunk

File: cows/al.cow [hunk 13]
```diff
@@ -5,84 +5,84 @@ $the_cow = <<EOC;
...
+                                              ..................................
```
Comment (AlMcKinlay) [inline cows/al.cow:26]: This line appears to be truncated rather than shrunk

File: cows/al.cow [hunk 14]
```diff
@@ -5,84 +5,84 @@ $the_cow = <<EOC;
...
+                                              ..................................
```
Comment (AlMcKinlay) [inline cows/al.cow:27]: This line appears to be truncated rather than shrunk

File: cows/al.cow [hunk 15]
```diff
@@ -5,84 +5,84 @@ $the_cow = <<EOC;
...
+                                             .,.................................
```
Comment (AlMcKinlay) [inline cows/al.cow:28]: This line appears to be truncated rather than shrunk

File: cows/al.cow [hunk 16]
```diff
@@ -5,84 +5,84 @@ $the_cow = <<EOC;
...
+                                             .,.................................
```
Comment (AlMcKinlay) [inline cows/al.cow:29]: This line appears to be truncated rather than shrunk

File: cows/al.cow [hunk 17]
```diff
@@ -5,84 +5,84 @@ $the_cow = <<EOC;
...
+                                             ...................................
```
Comment (AlMcKinlay) [inline cows/al.cow:30]: This line appears to be truncated rather than shrunk

File: cows/al.cow [hunk 18]
```diff
@@ -5,84 +5,84 @@ $the_cow = <<EOC;
...
+                                             ...................................
```
Comment (AlMcKinlay) [inline cows/al.cow:31]: This line appears to be truncated rather than shrunk

File: cows/al.cow [hunk 19]
```diff
@@ -5,84 +5,84 @@ $the_cow = <<EOC;
...
+                                             ...................................
```
Comment (AlMcKinlay) [inline cows/al.cow:32]: This line appears to be truncated rather than shrunk

File: cows/al.cow [hunk 20]
```diff
@@ -5,84 +5,84 @@ $the_cow = <<EOC;
...
+                                             .................................,,
```
Comment (AlMcKinlay) [inline cows/al.cow:33]: This line appears to be truncated rather than shrunk

File: cows/al.cow [hunk 21]
```diff
@@ -5,84 +5,84 @@ $the_cow = <<EOC;
...
+                                             .,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
```
Comment (AlMcKinlay) [inline cows/al.cow:34]: This line appears to be truncated rather than shrunk

File: cows/al.cow [hunk 22]
```diff
@@ -5,84 +5,84 @@ $the_cow = <<EOC;
...
+                                          ./(((/********///////(((((/**,,,,,**//
```
Comment (AlMcKinlay) [inline cows/al.cow:35]: This line appears to be truncated rather than shrunk

File: cows/al.cow [hunk 23]
```diff
@@ -5,84 +5,84 @@ $the_cow = <<EOC;
...
+                                          ,##%/*,**********//////***(##((((###(/
```
Comment (AlMcKinlay) [inline cows/al.cow:36]: This line appears to be truncated rather than shrunk

File: cows/al.cow [hunk 24]
```diff
@@ -5,84 +5,84 @@ $the_cow = <<EOC;
...
+                                           .//*,,,,,,,*****//(((//***/%%%%&%%/**
```
Comment (AlMcKinlay) [inline cows/al.cow:37]: This line appears to be truncated rather than shrunk

File: cows/al.cow [hunk 25]
```diff
@@ -5,84 +5,84 @@ $the_cow = <<EOC;
...
+                                           .**,,,,,,****/(#####((((/*(%#/*/#%/*/
```
Comment (AlMcKinlay) [inline cows/al.cow:38]: This line appears to be truncated rather than shrunk

File: cows/al.cow [hunk 26]
```diff
@@ -5,84 +5,84 @@ $the_cow = <<EOC;
...
+                                           ,/*,,,,,**//**((###(((((/*##*..,*#(**
```
Comment (AlMcKinlay) [inline cows/al.cow:39]: This line appears to be truncated rather than shrunk

File: cows/al.cow [hunk 27]
```diff
@@ -5,84 +5,84 @@ $the_cow = <<EOC;
...
+                                           ,*/,.,,********///////***(#/,..,,/#/*
```
Comment (AlMcKinlay) [inline cows/al.cow:40]: This line appears to be truncated rather than shrunk

File: cows/al.cow [hunk 28]
```diff
@@ -5,84 +5,84 @@ $the_cow = <<EOC;
...
+                                           ,,**,..,,*,,,******//***/#/,,,,,,,(#/
```
Comment (AlMcKinlay) [inline cows/al.cow:41]: This line appears to be truncated rather than shrunk

File: cows/al.cow [hunk 29]
```diff
@@ -5,84 +5,84 @@ $the_cow = <<EOC;
...
+                                            ..,**,,,,,,,,*****,,,*((*,,,,,,,,*/(
```
Comment (AlMcKinlay) [inline cows/al.cow:42]: This line appears to be truncated rather than shrunk

File: cows/al.cow [hunk 30]
```diff
@@ -5,84 +5,84 @@ $the_cow = <<EOC;
...
+                                            ....,,,*************//*,,,,,,,,,,,**
```
Comment (AlMcKinlay) [inline cows/al.cow:43]: This line appears to be truncated rather than shrunk

File: cows/al.cow [hunk 31]
```diff
@@ -5,84 +5,84 @@ $the_cow = <<EOC;
...
+                                            .......,,,,,,,,,,***,,,,,,,.,,,,,,,*
```
Comment (AlMcKinlay) [inline cows/al.cow:44]: This line appears to be truncated rather than shrunk

File: cows/al.cow [hunk 32]
```diff
@@ -5,84 +5,84 @@ $the_cow = <<EOC;
...
+                                            .......,,,,,,,,,,,,,*,,,,,,,,,,,,,,,
```
Comment (AlMcKinlay) [inline cows/al.cow:45]: This line appears to be truncated rather than shrunk

File: cows/al.cow [hunk 33]
```diff
@@ -5,84 +5,84 @@ $the_cow = <<EOC;
...
+                                             .......,,,...,,,,***,,,,,,,*****/(/
```
Comment (AlMcKinlay) [inline cows/al.cow:46]: This line appears to be truncated rather than shrunk

File: cows/al.cow [hunk 34]
```diff
@@ -5,84 +5,84 @@ $the_cow = <<EOC;
...
+                                             .......,...,,,,,*******((/*****////
```
Comment (AlMcKinlay) [inline cows/al.cow:47]: This line appears to be truncated rather than shrunk

File: cows/al.cow [hunk 35]
```diff
@@ -5,84 +5,84 @@ $the_cow = <<EOC;
...
+                                              ..,,,.,,,,,,,,,***********/*//****
```
Comment (AlMcKinlay) [inline cows/al.cow:48]: This line appears to be truncated rather than shrunk

File: cows/al.cow [hunk 36]
```diff
@@ -5,84 +5,84 @@ $the_cow = <<EOC;
...
+                                              ..,,,,,,,,,,,,********************
```
Comment (AlMcKinlay) [inline cows/al.cow:49]: This line appears to be truncated rather than shrunk

File: cows/al.cow [hunk 37]
```diff
@@ -5,84 +5,84 @@ $the_cow = <<EOC;
...
+                                              ..,,,,,,,,,,,****,,,,,,,,,,,,,****
```
Comment (AlMcKinlay) [inline cows/al.cow:50]: This line appears to be truncated rather than shrunk

File: cows/al.cow [hunk 38]
```diff
@@ -5,84 +5,84 @@ $the_cow = <<EOC;
...
+                                               .,,,,,,,,,,,**,,,**************/*
```
Comment (AlMcKinlay) [inline cows/al.cow:51]: This line appears to be truncated rather than shrunk

File: cows/al.cow [hunk 39]
```diff
@@ -5,84 +5,84 @@ $the_cow = <<EOC;
...
+                                               .,,,,,,,,,,,***(#//***,,,,,,,**//
```
Comment (AlMcKinlay) [inline cows/al.cow:52]: This line appears to be truncated rather than shrunk

File: cows/al.cow [hunk 40]
```diff
@@ -5,84 +5,84 @@ $the_cow = <<EOC;
...
+                                                .,,,,,,,,,,,****//((((/******/((
```
Comment (AlMcKinlay) [inline cows/al.cow:53]: This line appears to be truncated rather than shrunk

File: cows/al.cow [hunk 41]
```diff
@@ -5,84 +5,84 @@ $the_cow = <<EOC;
...
+                                                ..,,,,,,,,,,***,,,**************
```
Comment (AlMcKinlay) [inline cows/al.cow:54]: This line appears to be truncated rather than shrunk

File: cows/al.cow [hunk 42]
```diff
@@ -5,84 +5,84 @@ $the_cow = <<EOC;
...
+                                                 .,,,,,,,,,,*,,,,,,,,,**********
```
Comment (AlMcKinlay) [inline cows/al.cow:55]: This line appears to be truncated rather than shrunk

File: cows/al.cow [hunk 43]
```diff
@@ -5,84 +5,84 @@ $the_cow = <<EOC;
...
+                                                  .,,,,,,,,,,,,,,,,,,,**********
```
Comment (AlMcKinlay) [inline cows/al.cow:56]: This line appears to be truncated rather than shrunk

File: cows/al.cow [hunk 44]
```diff
@@ -5,84 +5,84 @@ $the_cow = <<EOC;
...
+                                                   .,,,,,,,,,,,,,,,,,,,*********
```
Comment (AlMcKinlay) [inline cows/al.cow:57]: This line appears to be truncated rather than shrunk

File: cows/al.cow [hunk 45]
```diff
@@ -5,84 +5,84 @@ $the_cow = <<EOC;
...
+                                                    .,,,,,,,,,,,,,,,,,,,,,,,,,,,
```
Comment (AlMcKinlay) [inline cows/al.cow:58]: This line appears to be truncated rather than shrunk

File: cows/al.cow [hunk 46]
```diff
@@ -5,84 +5,84 @@ $the_cow = <<EOC;
...
+                                                     .,,,,,,,,,,,,,,,,,,,,,,,,,,
```
Comment (AlMcKinlay) [inline cows/al.cow:59]: This line appears to be truncated rather than shrunk

File: cows/al.cow [hunk 47]
```diff
@@ -5,84 +5,84 @@ $the_cow = <<EOC;
...
+                                                       .,,,,,,,,,,,,,,,,,,,,,,,,
```
Comment (AlMcKinlay) [inline cows/al.cow:60]: This line appears to be truncated rather than shrunk

File: cows/al.cow [hunk 48]
```diff
@@ -5,84 +5,84 @@ $the_cow = <<EOC;
...
+                                                      .,,,,,,,,,,,,,,,,,,,,,,,,,
```
Comment (AlMcKinlay) [inline cows/al.cow:61]: This line appears to be truncated rather than shrunk

File: cows/al.cow [hunk 49]
```diff
@@ -5,84 +5,84 @@ $the_cow = <<EOC;
...
+                                                    ./%#*,,,,,,,***,************
```
Comment (AlMcKinlay) [inline cows/al.cow:62]: This line appears to be truncated rather than shrunk

File: cows/al.cow [hunk 50]
```diff
@@ -5,84 +5,84 @@ $the_cow = <<EOC;
...
+                                               ..*####*,.,,,*,*,,,**************
```
Comment (AlMcKinlay) [inline cows/al.cow:63]: This line appears to be truncated rather than shrunk

File: cows/al.cow [hunk 51]
```diff
@@ -5,84 +5,84 @@ $the_cow = <<EOC;
...
+                                           .,*##%###/,....,,,,*****,************
```
Comment (AlMcKinlay) [inline cows/al.cow:64]: This line appears to be truncated rather than shrunk

File: cows/al.cow [hunk 52]
```diff
@@ -5,84 +5,84 @@ $the_cow = <<EOC;
...
+                                       .,/#%%#%%%%%(,......,,,,*****************
```
Comment (AlMcKinlay) [inline cows/al.cow:65]: This line appears to be truncated rather than shrunk

File: cows/al.cow [hunk 53]
```diff
@@ -5,84 +5,84 @@ $the_cow = <<EOC;
...
+                                  .,*/#%%%%%%%&%%%%/,.......,,,,,***************
```
Comment (AlMcKinlay) [inline cows/al.cow:66]: This line appears to be truncated rather than shrunk

File: cows/al.cow [hunk 54]
```diff
@@ -5,84 +5,84 @@ $the_cow = <<EOC;
...
+                             .,*(#%%%%%%%%%%%&&%&%%(,.........,,,,,,************
```
Comment (AlMcKinlay) [inline cows/al.cow:67]: This line appears to be truncated rather than shrunk

File: cows/al.cow [hunk 55]
```diff
@@ -5,84 +5,84 @@ $the_cow = <<EOC;
...
+                         .*(##%%%%%%%%%%%%%%&&&&&&&%*...........,.,,,,,,,,,,,,,,
```
Comment (AlMcKinlay) [inline cows/al.cow:68]: This line appears to be truncated rather than shrunk

File: cows/al.cow [hunk 56]
```diff
@@ -5,84 +5,84 @@ $the_cow = <<EOC;
...
+                     .,/##%%%%%%%%%&&&&%&%&%%&&&&&&*,......,,,,.,.,.,,,,,,,,,,,,
```
Comment (AlMcKinlay) [inline cows/al.cow:69]: This line appears to be truncated rather than shrunk

File: cows/al.cow [hunk 57]
```diff
@@ -5,84 +5,84 @@ $the_cow = <<EOC;
...
+                ..*/##%%%%%%&&&&&&&&&&&&&&&&&&&&&&&&%(*,,.......,.,.,...,,,,,,,,
```
Comment (AlMcKinlay) [inline cows/al.cow:70]: This line appears to be truncated rather than shrunk

File: cows/al.cow [hunk 58]
```diff
@@ -5,84 +5,84 @@ $the_cow = <<EOC;
...
+            .,(##%%%%%&%%%&&&&&&&&&&&&&&&%&&&&&&&&&&&%(*,,,.....,.,.,.,,,.,,,,,,
```
Comment (AlMcKinlay) [inline cows/al.cow:71]: This line appears to be truncated rather than shrunk

File: cows/al.cow [hunk 59]
```diff
@@ -5,84 +5,84 @@ $the_cow = <<EOC;
...
+        .,(##%%%%%&&&&&&&&&&&&&&&&&&&&%&&&&&&&&&&&&&&&%(*,,,,,..,,,,,,,.,,,...,,
```
Comment (AlMcKinlay) [inline cows/al.cow:72]: This line appears to be truncated rather than shrunk

File: cows/al.cow [hunk 60]
```diff
@@ -5,84 +5,84 @@ $the_cow = <<EOC;
...
+     .*(##%%%%%%%%&%&%&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&*,,,,,,,.................
```
Comment (AlMcKinlay) [inline cows/al.cow:73]: This line appears to be truncated rather than shrunk

File: cows/al.cow [hunk 61]
```diff
@@ -5,84 +5,84 @@ $the_cow = <<EOC;
...
+   ./#%%%%%%%%%%&%&%&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&%/,,,,.................
```
Comment (AlMcKinlay) [inline cows/al.cow:74]: This line appears to be truncated rather than shrunk

File: cows/al.cow [hunk 62]
```diff
@@ -5,84 +5,84 @@ $the_cow = <<EOC;
...
+ *(#%%%%%%%%&%&&&&&&&&&&&&&&&&&%&&&&&&&&&&&&&&&&&&&&&&&&&&&%/,,.................
```
Comment (AlMcKinlay) [inline cows/al.cow:75]: This line appears to be truncated rather than shrunk

File: cows/al.cow [hunk 63]
```diff
@@ -5,84 +5,84 @@ $the_cow = <<EOC;
...
+(###%%%%%%&%&%%%&&%%&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&%(,................
```
Comment (AlMcKinlay) [inline cows/al.cow:76]: This line appears to be truncated rather than shrunk

File: cows/al.cow [hunk 64]
```diff
@@ -5,84 +5,84 @@ $the_cow = <<EOC;
...
+%#%%%%%%%&&%%%&%&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&%%(*,,,.........,
```
Comment (AlMcKinlay) [inline cows/al.cow:77]: This line appears to be truncated rather than shrunk

File: cows/al.cow [hunk 65]
```diff
@@ -5,84 +5,84 @@ $the_cow = <<EOC;
...
+%%%%&&%%&&&&&%&&&&&&&&&&&&&&&&&%&&&%&%&%&&&%&&&%&%&%&%&&&&&&&&&&&%&&&&&%%%%%%%&&
```
Comment (AlMcKinlay) [inline cows/al.cow:78]: This line appears to be truncated rather than shrunk

File: cows/al.cow [hunk 66]
```diff
@@ -5,84 +5,84 @@ $the_cow = <<EOC;
...
+%%%&&%%&&&&&&%&&%%&%&%%%&%&%&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&
```
Comment (AlMcKinlay) [inline cows/al.cow:79]: This line appears to be truncated rather than shrunk

File: cows/al.cow [hunk 67]
```diff
@@ -5,84 +5,84 @@ $the_cow = <<EOC;
...
+%%&&&%%&&&&%&%&&&&&&&&&%&&&&&&&%%%%%&&&%&&&&&%&%&&&&&&&%&&&&&&&&&&&&&&&&&&&&&&&&
```
Comment (AlMcKinlay) [inline cows/al.cow:80]: This line appears to be truncated rather than shrunk

File: cows/al.cow [hunk 68]
```diff
@@ -5,84 +5,84 @@ $the_cow = <<EOC;
...
+&%&&&%&&&&&&&%&&&&&%&%&%&&&&&&&&&&&&&&&&&%&&&%&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&
```
Comment (AlMcKinlay) [inline cows/al.cow:81]: This line appears to be truncated rather than shrunk

File: cows/al.cow [hunk 69]
```diff
@@ -5,84 +5,84 @@ $the_cow = <<EOC;
...
+&&&&%%&&&&&&&&&%&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&%%&&&&%&&&&&&&&&&&&&&&&&&&&&&&&&&&
```
Comment (AlMcKinlay) [inline cows/al.cow:82]: This line appears to be truncated rather than shrunk

File: cows/al.cow [hunk 70]
```diff
@@ -5,84 +5,84 @@ $the_cow = <<EOC;
...
+%&@&&&&&&%%%&%&&&&&&&%&&&&&%&%&&&&&&&&&&&&&&&&&&&&&&&&&&&%&&&&&&&&&&&&&&&&&&&&&&
```
Comment (AlMcKinlay) [inline cows/al.cow:83]: This line appears to be truncated rather than shrunk

File: cows/al.cow [hunk 71]
```diff
@@ -5,84 +5,84 @@ $the_cow = <<EOC;
...
+%&&&&&&%&&&&&%%%%%&%&&&&&&%%&&&%&&&&&&&&&&&&&&%%&&&&&&&&&%&&&&&&&&&&&&&&&&&&&&&&
```
Comment (AlMcKinlay) [inline cows/al.cow:84]: This line appears to be truncated rather than shrunk

File: cows/al.cow [hunk 72]
```diff
@@ -5,84 +5,84 @@ $the_cow = <<EOC;
...
+%&&&&&&&%&&%%%%%%%&&&%%%%%%%&%&&&&&&&&&&&&&&&%&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&
```
Comment (AlMcKinlay) [inline cows/al.cow:85]: This line appears to be truncated rather than shrunk

File: cows/al.cow [hunk 73]
```diff
@@ -5,84 +5,84 @@ $the_cow = <<EOC;
...
+&&&&&&%&&%%%%%%%&%&&&&%%&&&%&&&&&&&&&&&&&&&%&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&
```
Comment (AlMcKinlay) [inline cows/al.cow:86]: This line appears to be truncated rather than shrunk

**Review by AlMcKinlay — CHANGES_REQUESTED** [review]
Looks good, just a few issues.

**Comment by AlMcKinlay** [comment]
I will defer to @amleaver as to whether or not this PR goes in as-is, as he was the original requester of the feature, and this wouldn't create a large maintenance burden.

**Comment by chooban** [comment]
I don't think using a smaller font would help with readability.

**Comment by amleaver** [comment]
This is garbage, if alsay is to succeed as an OS project then we have to do better

**Pull request closed without merging**